build.gradle

```diff
@@ -49,7 +49,9 @@ dependencies {
                     "com.sun.jersey:jersey-core:1.19",
                     "com.sun.jersey.contribs:jersey-multipart:1.19",
                     "org.apache.taglibs:taglibs-standard-impl:1.2.5",
-                    "org.jsoup:jsoup:1.9.2"
+                    "org.jsoup:jsoup:1.9.2",
+                    "org.apache.poi:poi:3.9",
```

Comment:
I think it should be better if we use the latest version since it's stable already, unless we find any reason why 3.9 is better.